jekyll/_cci2/skip-build.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: classic-docs
+title: Skipping a Build
```

Comment:
We do want the additional metadata here as well, in case we go back to using the jekyll nav